fusillade/clouddirectory.py

```diff
@@ -1735,25 +1735,23 @@ def get_users_page(self, next_token=None, per_page=None) -> Tuple[Dict, str]:
             next_token=next_token)
         return {'users': results}, next_token
 
-    def add_users(self, users: List['User']) -> None:
+    def add_users(self, users: List[str]) -> None:
         if users:
-            User.exists([user.name for user in users])
-            operations = [i for i in itertools.chain(*[user.add_groups([self.name], False) for user in users])]
+            operations = [i for i in itertools.chain(*[User(user).add_groups([self.name], False) for user in users])]
```

Comment:
I think this would be clearer as

```suggestion
            operations = list(itertools.chain(*[User(user).add_groups([self.name], False) for user in users]))
```

It might also be clearer to use `itertools.from_iterable` but that's personal preference.

Reply:
Looks cleaner. TY